README.md

```diff
@@ -2,6 +2,35 @@
 
 > Our guidelines for how we do Scrum
 
+## Development
+
+Development of an implementation of a backlog item is carried out in accordance to [GitHub Flow](https://guides.github.com/introduction/flow/) with some additional requirements and tooling support. One of the primary requirements of the development workflow is the use of [protected branches](https://help.github.com/articles/configuring-protected-branches/) in order to enforce the workflow and ensure the integrity of the main product branches.
+
+### Workflow
+
+1.  __Create a topic branch__  
+    All work is to be carried out on topic branches to ensure isolation. Make sure that the name of your branch describes the work you'll be doing and that it's written in kebab-case (eg. `my-topic-branch`).
```

Comment:
Branch prefixes work well with the somewhat more complex [gitflow branching model](http://nvie.com/posts/a-successful-git-branching-model/), but I've found that they tend to overcomplicate GitHub Flow.